_pages/do-i-need-clearance/index.md

```diff
@@ -39,30 +63,11 @@ Asking for information to be sent to the government, including:
 - Voluntary commercial transactions such as information necessary and solely used for payment and delivery of an item.
 - Information asked for or received in connection to a public hearing or meeting.
 
-#### Information at public meetings and online
-
-The discussion and conversation that takes place at a public meeting has long been exempt from the PRA and using online interactive meeting tools such as public conferences calls, webinars, discussions board and forums, and chat sessions are considered the online equivalent of in-person public meetings and do not need PRA clearance.
-
 Some information collections are generally not subject to the PRA, including: federal investigations and civil actions, antitrust actions, and intelligence activities.
 
-### Who are you collecting information from?
-
-#### Members of the public
-
-Generally, this means people or groups outside of the federal government. Some groups that are considered members of the public include:
-
-- Individual people (including federal contractors)
-- Businesses and associations
-- State, tribal, and local governments
-
-If you’re only collecting information from federal employees or military personnel as part of their job, then you don’t need PRA clearance. If the information isn’t part of their work-related duties, you will need PRA clearance.
-
-#### Voluntary or Mandatory Collection
+#### Information at public meetings and online
 
-Regardless if your collection is voluntary, (i.e. the public is not required by law to provide information), or mandatory, the Paperwork Reduction Act treats the collections the same.
+The discussion and conversation that takes place at a public meeting has long been exempt from the PRA and using online interactive meeting tools such as public conferences calls, webinars, discussions board and forums, and chat sessions are considered the online equivalent of in-person public meetings and do not need PRA clearance.
```

Comment:
This sentence feels pretty long. Wondering if we could break it up, or if it makes sense to make a list of online interactive meeting tools that don't need PRA clearance for easier scanning.